bluemira/equilibria/opt_objectives.py

```diff
@@ -115,6 +125,62 @@ def regularised_lsq_objective(vector, grad, scale, a_mat, b_vec, gamma):
     return fom
 
 
+def minimise_coil_currents(vector, grad):
```

Comment:
Only comment on why you'd want to minimise the absolute currents rather than their squares (which will have better behaved gradients)?